frontend/components/organiser/event/forms/EventDrilldownFormsPage.tsx

```
@@ -177,21 +176,27 @@ const EventDrilldownFormsPage = ({ eventId, eventMetadata }: EventDrilldownForms
+        <h1 className="text-2xl font-extrabold mb-1">Form Responses</h1>
+        {formTitle && <p className="text-sm text-gray-400 line-clamp-1">Form: {formTitle}</p>}
+      </div>
+      <div className="flex items-center gap-2">
+        {formResponses.length > 0 && (
+          <DownloadCsvButton data={csvData} headers={csvHeaders} filename={`FormResponses_${eventId}.csv`} />
         )}
+        <button
+          onClick={() => setIsAddFormResponseDialogOpen(true)}
+          className="inline-flex justify-center rounded-md bg-organiser-dark-gray-text px-2 md:px-4 py-1.5 md:py-2 text-sm font-medium text-white hover:bg-black/30 focus:outline-none focus-visible:ring-2 focus-visible:ring-white/75 hover:cursor-pointer"
+        >
+          <PlusIcon className="md:mr-2 h-5 w-5" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)